app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :integer          not null, primary key
+#  email                  :string(32)       not null
+#  display_name           :string(128)      not null
+#  password               :string(255)      default(""), not null
+#  admin                  :boolean          default(FALSE), not null
+#  disabled               :boolean          default(FALSE)
+#  created_at             :datetime         not null
+#  modified_at            :datetime
+#  biography              :text(65535)
+#  slides_count           :integer          default(0)
+#  encrypted_password     :string(255)      default(""), not null
+#  reset_password_token   :string(255)
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  sign_in_count          :integer          default(0), not null
+#  current_sign_in_at     :datetime
+#  last_sign_in_at        :datetime
+#  current_sign_in_ip     :string(255)
+#  last_sign_in_ip        :string(255)
+#  avatar_file_name       :string(255)
+#  avatar_content_type    :string(255)
+#  avatar_file_size       :integer
+#  avatar_updated_at      :datetime
+#
+
+module Api
+  module V1
```

Comment:
__[reek]__
* [UncommunicativeModuleName] has the name 'V1' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Module-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->